frontend/src/entry-forms/Transaction.svelte

```diff
@@ -21,12 +21,18 @@
     entry.postings = entry.postings.concat(emptyPosting());
   }
 
+  $: suggestions = $accounts.filter(
+    (value) => $account_details[value].close_date >= new Date(entry.date)
```

Comment:
This does the string->date conversion for every account, which is unnecessary, doing so once before the loop would be more efficient.

Reply:
Also, `$account_details[value]` might be undefined.